docs/python/generators/examples_helpers.py

```
@@ -86,6 +86,105 @@ def _find_unquoted_word_on_nontrivial_line(word: str, text: str) -> bool:
+    
+    For every provided name, if a page entry is found, also add any other
+    mapping keys that point to the same page (aliases created by
+    <!-- aui:index_alias ... -->). Aliases are matched by title only.
+    """
+    out = set([n for n in (names_in or []) if n])
+    try:
+        from docs.python.generators import index as docs_index
+        for n in list(out):
+            try:
+                entry = docs_index.find_page(n)
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)